Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -148,6 +148,8 @@ def condition(self, fragment, stepList, key, hasHarvest):
     def preventReuse(self, stepName, stepDict, k):
         if "Sim" in stepName:
             stepDict[stepName][k] = None
+        if "Gen" in stepName:
```

Comment:
`elif` would have been more appropriate, but it doesn't really matter here

Reply:
Thanks @perrotta 
I first use `elif` but see some unexpected issue. I can investigate more in the next PR, already plan to update the prod-like workflow for FastSim, and this point can be checked with detail.

Reply:
Wrong comment before, deleted. Yes, this can change to `elif`.